org-cliplink-string.el

```diff
@@ -5,14 +5,15 @@
 (defun org-cliplink-join-string (ss)
   (mapconcat #'identity ss " "))
 
+(setq org-cliplink-ellipsis "...")
```

Comment:
I think this kind of customization variables should be defined with [defcustom](https://www.gnu.org/software/emacs/manual/html_node/eintr/defcustom.html). Just like [org-cliplink-max-length](https://github.com/rexim/org-cliplink/blob/82402cae7e118d67de7328417fd018a18f95fac2/org-cliplink.el#L379-L384) variable.

Reply:
Fixed. Is it alright like this?